rclcpp/include/rclcpp/service.hpp

```diff
@@ -33,6 +35,7 @@
 #include "rmw/error_handling.h"
 #include "rmw/rmw.h"
 
+
```

Comment:
nitpick: minimize vertical whitespace and also avoid pure whitespace changes in pr's